plugins/src/main/java/com/google/fhir/gateway/plugin/ListAccessChecker.java

```diff
@@ -227,6 +228,22 @@ private AccessDecision processPatch(RequestDetailsReader requestDetails) throws
     return checkNonPatientAccessInUpdate(requestDetails, RequestTypeEnum.PATCH);
   }
 
+  private AccessDecision processDelete(RequestDetailsReader requestDetails) {
+    // We don't support deletion of List resource used as an access list for a user.
```

Comment:
In general, I think we should not let deletion of any `List` resource without a patient parameter, should we?

Reply:
yes right.
It is same as deletion of any other resource. Fixed this.